Sources_v3/StreamChatUI/ChatChannel/ChatVC.swift

```diff
@@ -150,7 +150,7 @@ open class ChatVC<ExtraData: ExtraDataTypes>: ViewController,
     public func chatMessageListVC(_ vc: ChatMessageListVC<ExtraData>, didTapOnRepliesFor message: _ChatMessage<ExtraData>) {}
 
     public func chatMessageListVC(_ vc: ChatMessageListVC<ExtraData>, didTapOn attachment: _ChatMessageAttachment<ExtraData>) {
-        debugPrint(attachment)
+        messageList.router.showPreview(for: attachment)
```

Comment:
Violation of encapsulation principle.

If you want handle attachments previews inside message list controller (you put it inside his router), then we need to drop this delegate method.

Otherwise `showPreview` should be in `ChatVC.router`

Reply:
fixed in ebc4d74abc5f051b609a29fe3eb22faf4fa8109b